### ISSUE TITLE
Color System: incorrect naming

### ISSUE THREAD

**Issue by hicucu** [primefaces/primereact#4228]
### Describe the bug

It seems that the content of the document does not match the actual information. As I understand it, the variable names "--text-secondary-color" and "--mask-bg" mentioned in the "Overview of the Color System" are different from the actual names. If that is the case, the correct names in English are:

    "--text-secondary-color" should be "--text-color-secondary".
    "--mask-bg" should be "--maskbg".
![1](https://user-images.githubusercontent.com/48123604/230242416-6e581cf1-f372-4bc2-a845-f65b4d7b5dae.png)
![2](https://user-images.githubusercontent.com/48123604/230242423-3818f2c4-e102-4415-826b-9ae92d77cbbf.png)

### PrimeReact version

9.2.2

### React version

18.x

### Language

TypeScript

### Build / Runtime

Create React App (CRA)

### Steps to reproduce the behavior

To maintain consistency, it is recommended to use "--text-secondary-color" consistently throughout the document. Additionally, it is suggested to use "--mask-bg" instead of "--maskbg" as it follows a more consistent naming convention with a dash separating the words.

In summary, the recommended variable names in English are:

    "--text-secondary-color"
    "--mask-bg"

**Comment by JesseKoldewijn** [comment]
I mean, from what I see just from the image provided it sound more logical to me to have the secondary text var be as follows:
```--secondary-color-text```
Because as you can see the primary text color is called:
```--primary-color-text```
